web/controllers/user_controller.ex

```diff
@@ -11,6 +11,22 @@ defmodule CoursePlanner.UserController do
     render(conn, "index.html", users: Users.all())
   end
 
+  def show(%{assigns: %{current_user: %{role: role}}} = conn, %{"id" => id}) do
+    if role === "Coordinator" do
```

Comment:
I will use the permission.ex if I can find out how can I add rules to that. Yesterday I spent relatively long time to replace that condition, but I still couldn't do that. I can't tell if it's necessary to hide the user show page anyway.